Resolutions/HyruleWarriors_Coop_AspectRatio/patches.txt

```diff
@@ -0,0 +1,7 @@
+[ProjectZ_219v208]
+moduleMatches = 0xE159AE91
+
+0x1009609C = .float $ratio
+0x10096124 = .float $ratio
+0x10096948 = .float $ratio
+0x10096974 = .float $ratio
```

Comment:
Are you sure we need to change all these values? I feel like two of them would be used just for the co-op mode.

Reply:
I didn't check, but I got those values from the resolution pack, and that uses all 4 of them.

Reply:
Did you resolve it because you checked to make sure all 4 changes were necessary?